crates/forge/tests/integration/gas.rs

```diff
@@ -697,11 +701,17 @@ fn l1_handler_cost() {
 
     let result = run_test_case(&test);
     assert_passed(&result);
-    // FIXME these values changed
+    // TODO(#2960)
```

Comment:
Let's extend this comment to say that the calculations below are based on code from blockifier and doesn't match the docs.

I know it says that in the issue, but it will be useful to have it here as well in case this stays unresolved for longer